### ISSUE TITLE
exception "No compatible MPEG-4 encoder found" raised when codec is mp4

### ISSUE THREAD

**Issue by hsafavi** [baSSiLL/SharpAvi#8]
i want to record my screen. but when i use mp4 or xvid and etc this exception is raised "No compatible MPEG-4 encoder found". how can i use this codecs?

**Comment by baSSiLL** [comment]
**Mpeg4VideoEncoderVcm** class uses MPEG-4 encoders installed in the system. So, to be able to do MPEG-4 encoding, you should install either of the codecs listed in docs

https://github.com/baSSiLL/SharpAvi/wiki/Using-Video-Encoders#creating-video-encoder

**Comment by therealsquidgee** [comment]
Hi @baSSiLL I downloaded the encoder you mentioned in the documentation but I am still receiving this exception. Any ideas why that might be? Great library btw!

**Comment by baSSiLL** [comment]
Hi @therealsquidgee 
Probably mismatching bitness between codec and program. For instance, you've installed 32-bit codec only and run your program as 64-bit process.

**Comment by husk3rdu** [comment]
Hi baSSiLL,

Same issue here on Windows 7 (64b). I've installed x264vfw for 32 and 64. A GetAvailableCodecs returns: 

x264vfw - H.264/MPEG-4 AVC codec

But i get:

Unhandled Exception: System.InvalidOperationException: Encoder operation returned an error: -100.
   at SharpAvi.Codecs.Mpeg4VideoEncoderVcm.CheckICResult(Int32 result)
   at SharpAvi.Codecs.Mpeg4VideoEncoderVcm.StartCompression()
   at SharpAvi.Codecs.Mpeg4VideoEncoderVcm..ctor(Int32 width, Int32 height, Double fps, Int32 frameCount, Int32 quality, FourCC[] codecPreference)
   at SharpAvi.Codecs.EncodingStreamFactory.<>c__DisplayClass2.<AddMpeg4VideoStream>b__0()
   at System.Windows.Threading.DispatcherOperation`1.InvokeDelegateCore()
   at System.Windows.Threading.DispatcherOperation.InvokeImpl()

I compiled it in 32/64 with the same result. By the way, it works on W10...
